server/api/stream.go

```diff
@@ -140,18 +140,18 @@ func LogStreamSSE(c *gin.Context) {
 	repo := session.Repo(c)
 	_store := store.FromContext(c)
 
-	// // parse the build number and job sequence number from
+	// // parse the pipeline number and job sequence number from
```

Comment:
@anbraten could this be the issue we face with dubble entry on side-bar, as we tread pipeline with different sequence as new one?

(unrelated to this pull)